### DISCUSSION TITLE
Error when I try to read a file using Capacitor in Android 11

### DISCUSSION THREAD

**Discussion by Gabriel-des** [ionic-team/capacitor#5033] [Q&A]
I'm trying for a few days to read some files in External Storage, but without success so far because of the new Android 11 rules. One of the main functions of my app is related to import files from the device's FileSystem for obtain and manipulate the data inside this file.

I've already tried perform the import in several ways, using the full path or abbreviated and import in differents directories, but without success. Finally I tried to import the file from Scoped Storage, but even so I couldn't read a file.

Not knowing what else to do, I made a question on [Stack Overflow](https://stackoverflow.com/questions/69131247/error-when-i-try-to-read-a-file-using-capacitor-in-android-11), but I didn't get a helpfull answers so far. You can check the question there to see more detailed attempts and functions used.

My app already exists and works without problems, but targeting API 29, and now I need it to be compliant with API 30 standards due to the changes in the Play Store.

In short, I need some help to enable the import of user files, even if it is in a specific folder. 

After for many mistakes, attempts, researches and failed answers I decided to appeal to this community. You are my new(and last) hope.

**Comment by robingenz** [comment]
I had a similar problem. The solution that worked very well for me was to use a File Picker plugin, which can read the file data directly, because the plugin has all the necessary rights at this time. Of course, my solution only works if the user actively selects the file via the File Picker plugin. Maybe this will help you or give you new ideas.
